packages/idos-sdk-js/package.json

```diff
@@ -10,13 +10,13 @@
   "files": ["dist"],
   "exports": {
     ".": {
-      "import": "./dist/idos-sdk.js",
-      "types": "./dist/idos-sdk.d.ts"
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
     }
   },
   "scripts": {
-    "dev": "vite build --mode development --watch",
-    "build": "vite build",
+    "dev": "dotenvx run -f .env.local -- tsup --watch",
+    "build": "dotenvx run -- tsup",
```

Comment:
I've maybe staple this to `.env.production`, just to make it terribly obvious what we're doing.